packages/reputation-miner/ReputationMiner.js

```diff
@@ -106,6 +102,18 @@ class ReputationMiner {
     this.gasPrice = ethers.utils.hexlify(20000000000);
     const repCycle = await this.getActiveRepCycle();
     await this.updatePeriodLength(repCycle);
+
+    // NB some technical debt here with names. The minerAddress arg passed in on the command line, which
+    // has been used for realWallet and is where transactions will be signed from may or may not be acting
+    // on behalf of another address via delegate mining. So check which address we're actually mining for.
+    const signingAddress = await this.realWallet.getAddress();
+    const delegator = await this.colonyNetwork.getMiningDelegator(signingAddress);
+    if (delegator === ethers.constants.AddressZero){
+      this.minerAddress = signingAddress;
+    } else {
+      this.minerAddress = delegator;
+    }
+    console.log("Mining on behalf of ", this.minerAddress)
```

Comment:
Nitpick but this style of string construction automatically inserts a space between arguments. I would slightly prefer the string template approach, but not really a big deal.